### DISCUSSION TITLE
Windows 11下受限帳戶不能顯示候選（僅顯示黑色區塊）

### DISCUSSION THREAD

**Discussion by LearZhou** [rime/home#1075] [General]
有一臺新機器，裝的是Windows 11，發現中州韻安裝之後，在管理員權限帳戶下能正常使用，但在受限帳戶下使用不能顯示候選列表，僅顯示黑色區塊。

起初我以爲是某些目錄權限問題，但嘗試將一些目錄的寫入權限給受限帳戶，也未能成功。

請問可有暫時的辦法能夠解決這個問題麼？

現在我就是瞎摸打字出來的，太難受了。

**Comment by LearZhou** [comment]
最後是這麼解決的，先把平時用的帳戶設定為管理員，然後用這個帳戶安裝中州韻，重啟安裝完畢。

此時再將平時使用帳戶的權限調低為普通用戶，就可以了。供大家參考。

**Comment by LearZhou** [comment]
但問題還是存在的，比如若有 2 個以上普通帳戶，那該怎麼辦？

**Comment by lotem** [comment]
能正常輸入，那可能只是缺少 `weasel.yaml` 裏的默認界面風格設置。
以用戶身份執行一次「重新部署」程序，能成功嗎。
